pandas/formats/format.py

```diff
@@ -873,6 +886,9 @@ def get_col_type(dtype):
                             compat.string_types):  # pragma: no cover
             raise AssertionError('column_format must be str or unicode, not %s'
                                  % type(column_format))
+        multicolumn_format = self.multicolumn_format
+        if multicolumn_format is None:
+            multicolumn_format = get_option("display.latex.multicolumn_format")
```

Comment:
I think it is enough that it is only checked in `DataFrame.to_latex`. A user should in principle never call the Formatter.to_latex directly

(it seems that we are a bit inconsistent here, as eg longtable and escate are only checked in DataFrame.to_latex)